mmselfsup/__init__.py

```diff
@@ -51,10 +52,17 @@ def digit_version(version_str: str, length: int = 4):
 mmcv_maximum_version = '1.5.0'
```

Comment:
`1.5.0` -> `1.6.0`
Besides, mmcls 0.21.0 requires mmcv >= 1.4.2, I think we also need update the mmcv minimum version

Reply:
As suggested, the minimum version of mmcv should be fixed.